app/controllers/course/discussion/topics_controller.rb

```diff
@@ -4,6 +4,6 @@ class Course::Discussion::TopicsController < Course::ComponentController
                                                  class: Course::Discussion::Topic.name,
                                                  parent: false
   def index
-    @topics = @topics.page(page_param)
+    @topics = @topics.display_globally.ordered_by_updated_at.page(page_param)
```

Comment:
I remembered wrongly I think.. just checked the discussion, it's `displayed_globally`